agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -152,10 +155,15 @@
     }
 
     public static class Heartbeat {
-
         public long intervalSeconds = MINUTES.toSeconds(15);
     }
 
+    public static class Statsbeat {
+        public String connectionString = "InstrumentationKey=c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint=https://westus-0.in.applicationinsights.azure.com/"; //workspace-aistatsbeat
+        public long intervalSeconds = DEFAULT_STATSBEAT_INTERVAL;
+        public long featureIntervalSeconds = FEATURE_STATSBEAT_INTERVAL;
+    }
```

Comment:
are these configurations exposed just for testing? or do we need to document them / expect users to need them?

also, how will this work in gov clouds that cannot access public cloud?